### DISCUSSION TITLE
Есть кто?

### DISCUSSION THREAD

**Discussion by gr33nka** [MPSU/exAPS#19] [Основное]
## Давайте знакомиться

Привет!

Как вам идея открыть обсуждения? Как вам кажется, будет ли кто-то кроме преподавателей участвовать в этом?
Лично я надеюсь, что да

**Comment by SS4V4GE** [comment]
куда писать?)

**Reply by gr33nka** [reply]
А прям сюда и писать)

**Reply by gr33nka** [reply]
Считай это как форум для этого репозитория, можно создавать темы в разных категориях, обсуждать их, а при необходимости превращать в issues

**Reply by SS4V4GE** [reply]
вас понял

**Comment by SS4V4GE** [comment]
Как я и предсказывал... Никто даже не увидел этого диалога

**Reply by gr33nka** [reply]
Ну так давай, привлекай там тусовку, раскидай ссылки, расскажи как тут весело)
Появилось еще несколько тем в обсуждениях, может у кого-то найдется что прокомментировать
